src/Model/ROI.py

```diff
@@ -715,9 +717,63 @@ def transform_rois_contours(axial_rois_contours):
                         sagittal_rois_contours[name][contour[i][0]] = [[]]
                         sagittal_rois_contours[name][contour[i][0]][0]\
                             .append([contour[i][1], slice_ids[slice_id]])
+
+    coronal_rois_contours = convert_coordinates_map_to_polygon_of_rois(
+        coronal_rois_contours)
+    sagittal_rois_contours = convert_coordinates_map_to_polygon_of_rois(
+        sagittal_rois_contours)
     return coronal_rois_contours, sagittal_rois_contours
 
 
+def convert_coordinates_map_to_polygon_of_rois(contours_map):
+    polygon_dict = {}
+    for name, contours_dict in contours_map.items():
+        polygon_dict[name] = {}
+        for slice_id, contour_array in contours_dict.items():
+            roi_array = contour_array[0]
+            hull_list = calculate_concave_hull_of_points(roi_array, alpha=0)
+            polygon_dict[name][slice_id] = hull_list
+    return polygon_dict
+
+
+def calculate_concave_hull_of_points(pixel_coords, alpha):
+    """
+        Return the alpha shape of the highlighted pixels using the alpha
+        entered by the user.
+        :param pixel_coords: the coordinates of the contour pixels
+        :return: List of lists of points ordered to form polygon(s).
+        """
+    # Get all the pixels in the drawing window's list of highlighted
+    # pixels, excluding the removed pixels.
+    target_pixel_coords = [(item[0] + 1, item[1] + 1) for item in
+                           pixel_coords]
+    # Calculate the concave hull of the points.
+    # alpha = 0.95 * alphashape.optimizealpha(points)
+    hull = target_pixel_coords
+    try:
+        hull = alphashape(target_pixel_coords, alpha)
+    except QhullError:
+        pass
+    polygon_list = []
+    if isinstance(hull, Polygon):
+        polygon_list.append(hull_to_points(hull))
+    elif isinstance(hull, MultiPolygon):
+        for polygon in hull:
+            polygon_list.append(hull_to_points(polygon))
+    return polygon_list
+
+
+# Convert hull to points
```

Comment:
Please turn comment into docstring for this function.

Reply:
done